Documentation admin panel (#1492)
kartoza/sawps: docs/src/administrator/manual/django-admin.md

Author: Okay

## Django Admin Layout

The Welcome page will be the first thing users see. It typically displays a welcome message and provides an overview of the available actions and options within the admin panel.

_![Django Admin Layout](./img/admin-2.png)_

In the top right corner of the admin panel, users will find a section that usually displays user's username. In this section, users have the following options:

1. **Welcome**: Displays welcome with admin name.

> 2. **View Site**: Clicking on this option will take users to the `SAWPS` website.

Reviewer: ```suggestion
2. **View Site**: Clicking on this option will take administrators to the `SAWPS` website.
```

> 3. **Change Password**: Clicking on this option allows users to change their admin password. Users will be prompted to provide their current password and set a new one.

Reviewer: ```suggestion
3. **Change Password**: Clicking on this option allows administrators to change their administration password. Administrators will be prompted to provide their current password and then set a new one.
```

    ![Change Password](./img/admin-8.png)

    **Input Fields**

>     1. **Old Password**: In this field, users should provide the current password associated with the user account. This is a mandatory field.

Reviewer: ```suggestion
    1. **Old Password**: In this field, administrators should provide the current password associated with their account. This is a mandatory field.
```

>     2. **New Password**: In this field, users should enter the new password they want to set for their account. The password should meet the following criteria:

Reviewer: ```suggestion
    2. **New Password**: In this field, administrators should enter the new password they want to set for their account. The password should meet the following criteria:
```

    * The password can't be too similar to other personal information.
    * The password must contain at least 12 characters.
    * The password can't be a commonly used password.
    * The password can't be entirely numeric.

>     * The password should include the following:
>     * Numeric character
>     * Uppercase letter
>     * Special character (@#%;)

Reviewer: ```suggestion
    * The password should include the following:
        * Numeric character
        * Uppercase letter
        * Special character (@#%;)
```

    * Ensure that the new password adheres to these requirements.

    3. **New Password Confirmation**: Re-enter the new password in this field to confirm it. It should match the password entered in the `New Password` field.

>     4. **Change My Password Button**: Once users have filled in the required information, they can click the `CHANGE MY PASSWORD` button to submit the form. If all the input is valid and the new password meets the criteria, the password for the user account will be updated.

Reviewer: ```suggestion
    4. **Change My Password Button**: Once administrators have filled in the required information, they can click the `CHANGE MY PASSWORD` button to submit the form. If all the input is valid and the new password meets the criteria, the password for their account will be updated.
```

> After successfully changing the password, the user will receive a confirmation message indicating that the password has been updated.

Reviewer: ```suggestion
After successfully changing the password, the administrator will receive a confirmation message indicating that the password has been updated.
```

> 4. **Log Out**: Clicking on this option will log the user out of the admin panel and return the user to the login page.

Reviewer: ```suggestion
4. **Log Out**: Clicking on this option will log the administrator out of the administration panel and return them to the login page.
```

    ![Logout admin page](./img/admin-9.png)

    1. **Log in again**: Clicking this option will open the login page.

> 5. **Recent Actions**: The `Recent Actions` section is typically located on the right side of the admin panel and provides a list of recent actions that have been performed within the admin interface. These actions may include additions, deletions, or changes to data in your project. Clicking on any of the available links will take a user to the detail of recent action.

Reviewer: ```suggestion
5. **Recent Actions**: The `Recent Actions` section is located on the right side of the administration panel and provides a list of recent actions that have been performed within the administration interface. These actions may include additions, deletions, or changes to data in the project. Clicking on any of the available links will take an administrator to the details of the recent action.
```

6. **Site Administration**: Site Administration this section provides links to various models and apps registered with the admin panel. It serves as the primary navigation point for managing a project's data. Clicking on any of the available links will take a user to the respective model's management page.

7. **Add**: This button is used to create a new item of the associated model. Clicking on `Add` link will open a form where users can enter the details of the new record.

8. **Change**: This button is used to edit an existing item from the list. When users click the `Change` button, they will be redirected to a page where they can see all the associate data of that model and can select data to change.

> * **Table**: To see data in a table, click on the table name.

Reviewer: Is this line still necessary? The same information is in line 72

Author: as we are doing the doc in tutorial style I think that having this line is okay

Reviewer: This is not a tutorial style document, this is the manual which is a page-by-page explanation. Please remove this line as the information is repeated in line 72 and it is more logical for it to be in line 72.

Author: Ah sorry, yes it is manual documentation. I will remove this line.
